snuba/datasets/cdc/cdcprocessors.py

```diff
@@ -15,10 +40,28 @@ def _process_commit(self, offset):
         pass
 
     def _process_insert(self, offset, columnnames, columnvalues):
```

Comment:
Probably should annotate these implemented methods with `-> snuba.writer.TableRow` while you're in here.